libminifi/include/core/PropertyType.h

```diff
@@ -138,7 +138,7 @@ class IntegerPropertyType : public PropertyType {
 
   [[nodiscard]] ValidationResult validate(const std::string &subject, const std::string &input) const override {
     try {
-      std::stoi(input);
+      (void) std::stoi(input);
```

Comment:
`[[maybe_unused]]` needs a variable (or function etc) name, so it would be
```c++
    [[maybe_unused]] auto unused = std::stoi(input);
```
Do you think that's better?

Reply:
Looking at it, not necessarily, so we can keep it as it is if you prefer, I don't have a preference.